test/lottotron.spec.js

```diff
@@ -92,6 +92,15 @@ describe('lottotron.js', function() {
       }
     }))
 
+
+    it('Should throw an error Should throw an "Error" object if the input param "maxNumber" is not finite value',
+      () => {
+        [NaN, +Infinity, -Infinity].forEach((notFinite) => {
+          assert.throws(() => new Lottotron(notFinite));
+        })
```

Comment:
Why do you write "should throw an Error object" but don't check the type of throw?